io.github.h8moss.progressive.yml

```
@@ -0,0 +1,74 @@
+app-id: io.github.h8moss.progressive
+runtime: org.gnome.Platform
+runtime-version: '49'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node20
+
+command: progressive
+finish-args:
+  # Wayland + X11 support for GUI
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  
+  # Filesystem access 
+  - --filesystem=xdg-documents  # User's Documents folder
+  - --filesystem=xdg-download   # User's Downloads folder
+  - --filesystem=xdg-videos     # User's Videos folder (For video series functionality)
+  
+  # GPU acceleration
+  - --device=dri
+
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/node20/bin
+  env:
+    CARGO_HOME: /run/build/progressive/cargo
+    XDG_CACHE_HOME: /run/build/progressive/flatpak-node/cache
+    npm_config_cache: /run/build/progressive/flatpak-node/npm-cache
+    npm_config_offline: "true"
+    LC_ALL: C.utf8
+
+modules:
+  - name: progressive
+    buildsystem: simple
+    build-options:
+      env:
+        # Tauri needs these
+        TAURI_SKIP_DEVSERVER_CHECK: 'true'
+        TAURI_DIST_DIR: /app
+    build-commands:
+      # Create binaries
+      - mkdir -p src-tauri/binaries
+      - cp /usr/bin/ffprobe src-tauri/binaries/ffprobe-x86_64-unknown-linux-gnu
+      - cp /usr/bin/ffprobe src-tauri/binaries/ffprobe-aarch64-unknown-linux-gnu
```

why should this be copied?

---

```suggestion
      - cp /usr/bin/ffprobe src-tauri/binaries/ffprobe-${FLATPAK_ARCH}-unknown-linux-gnu
```

but fixing the build system would be better.